src/components/open-position.jsx

```diff
@@ -63,10 +65,14 @@ const getOpenContractsInfo = () => {
 
 const OpenPosition = () => {
   onMount(async () => {
-    const active_account = JSON.parse(login_information?.active_account);
-    if (active_account) {
-      await wait("authorize");
-      getOpenContractsInfo();
+    try {
+      const active_account = JSON.parse(login_information?.active_account);
+      if (active_account) {
+        await wait("authorize");
```

Comment:
Can we wrap only the functions that make network calls in try & Catch. Coz if other code statements throw error, it might not be visible in console. This will be hard to debug during development

Reply:
updated